### ISSUE TITLE
Changes to raw breaking playbooks running against devel branch

### ISSUE THREAD

**Issue by juliakreger** [ansible/ansible#11119]
It appears https://github.com/ansible/ansible/commit/b94e2a1f4ee1631d311f6943f6653c391d5022de landed in the devel branch last night, and since then I've been unable to utilize the raw method.

I'm getting :

```
2015-06-02 01:39:19.415 | + ansible-playbook -vvvv -i inventory/localhost test-bifrost.yaml --syntax-check --list-tasks
2015-06-02 01:39:20.125 | ERROR! this task 'raw' has extra params, which is only allowed in the following modules: command, shell, script, include, include_vars, add_host, group_by, set_fact, meta
2015-06-02 01:39:20.125 | 
2015-06-02 01:39:20.125 | The error appears to have been in '/home/jenkins/workspace/gate-bifrost/playbooks/roles/bifrost-test-vm/tasks/main.yml': line 15, column 3, but may
2015-06-02 01:39:20.125 | be elsewhere in the file depending on the exact syntax problem.
2015-06-02 01:39:20.125 |
2015-06-02 01:39:20.125 | The offending line appears to be:
2015-06-02 01:39:20.125 | 
2015-06-02 01:39:20.125 | ---
2015-06-02 01:39:20.126 | - name: "Execute ping step to verify connectivity and login to the host.  If this fails, the configdrive may not have loaded."
2015-06-02 01:39:20.126 |   ^ here
```

Which is rooted in the following yaml, which makes total sense to me and seems reasonable as the target OS lacks SFTP and python:

```
- name: "Execute ping step to verify connectivity and login to the host.  If this fails, the configdrive may not have loaded."
  raw: hostname
  register: instance_hostname
```

Is this an intended change?  Am I doing something unreasonable/wrong?

**Comment by bcoca** [comment]
yesterday we switched to the new 'v2' engine, it seems that we are not handling the raw module properly

**Comment by bcoca** [comment]
try latest devel, fixed this via e251e701783ff053dc1d59a917bfaa9d788a2c6a

**Comment by juliakreger** [comment]
Thanks for the quick fix @bcoca and @jimi-c.  That got us past that issue.  We're running into other odd issues, enough that I'm going to push our jobs to using stable-1.9 for now.  Once I have it working, I'll open issues for devel as I find them.
